.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/packages/common" # Location of package manifests
```

Comment:
I think you are missing a directory or 2, but I guess this is on purpose?

Reply:
Yes, on purpose, just tbdex related packages for now because that's what I care about.